### ISSUE TITLE
CakePHP 4 support

### ISSUE THREAD

**Issue by Erwane** [Holt59/cakephp3-bootstrap-helpers#190]
I'm using this plugin since years now and i want to create a PR to support Cakephp4 with bootstrap4.

From which branch should i start please ?

**Comment by Holt59** [comment]
You should start with the https://github.com/Holt59/cakephp3-bootstrap-helpers/tree/v4.0.3 branch.

**Comment by Erwane** [comment]
Thanks, i'm on it.

I'm lloking the project name and versions and do you plan to rename it "cakephp-bootstrap-helpers" and think about a notation branchs for versions ?
`cake3-bootstrap3`, `cake3-bootstrap4`, `cake4-bootstrap4` ?

**Comment by Erwane** [comment]
in progress : https://github.com/Erwane/cakephp3-bootstrap-helpers/tree/cake4-compatibility

**Comment by Holt59** [comment]
@Erwane Thanks for all the work!

I have to check what implies a renaming of the Github project, I think Github can provide a redirection to the new project from the old URL but I'd have to check.

For the version, it's a bit more complicated unfortunately... For the branch names, there are currently no issues, but for composer to work properly there are some constraints on tagging (e.g., if you want `3.2.*` or something like that).

**Comment by Erwane** [comment]
Proposal :
- 3.3.x.y : cake 3, Bs 3
- 3.4.x.y : cake 3, Bs 4
- 4.4.x.y : cake 4, Bs 4

We only have "patch" (x) and "release" (y) numbers for new helpers or fixes.

In composer : 
`composer require holt59/cakephp-bootstrap:"^3.4.0"` for cake 3 with bootstrap 4

**Comment by Holt59** [comment]
@Erwane Yes, I think it's the best idea. And it does not break current versions which is a good thing.
